### ISSUE TITLE
How to improve validation accuracy 

### ISSUE THREAD

**Issue by chanduk123** [meownoid/face-identification-tpe#13]
Hello meownoid ,

would you please tell how to increase the validation accuracy during training.we applied MS-Celeba dataset and i am saving checkpoints for every 20 epoch and total epochs we put for training is 500 epoch, only one check point saved at 19th epoch with validation accuracy 0.008 and no other checkpoints were created at 40,60,80,100 epoch because of no change in validation accuracy.please help me to improve accuracy.

**Comment by chanduk123** [comment]
Hello meownoid ,
  the model which you given is giving less accuracy we need a better accuracy .would you please tell how to improve the accuracy and the better datasets for traning so that we expecting some improvement in accuracy.

**Comment by meownoid** [comment]
Repository is now archived, closing all issues.
